README.md

```diff
@@ -211,21 +211,21 @@ Puppeteer creates its own Chromium user profile which it **cleans up on every ru
 
 4. Stop test execution and use a debugger in browser
 
-  - Use `{devtools: true}` when launching Puppeteer:
+    - Use `{devtools: true}` when launching Puppeteer:
```

Comment:
Let's keep the indentation the same so there are no issues when we import the docs to developers.google.com.